ex11/part2/src/index.js

```diff
@@ -0,0 +1,28 @@
+import React, {useState} from "react";
+import ReactDOM from "react-dom";
+import Tb from "./timeModule";
+
+const App = () => {
+    const [textb, setTextb] = useState("0");
+
+    function timeChange(multyplyBy) {
+        return (e) => {
+            const newValue = Number(e.target.value * multyplyBy);
+            setTextb(newValue);
+        }
+    }
+
+    return (
+        <div>
+            <h1>ex 11 part 2</h1>
+            sec: <Tb text={textb} seter={timeChange(1)} divBy={1}/>
```

Comment:
The variable/prop names are no clear enough.
Try to give them longer and more descriptive names.